src/components/Name/RecordItem.tsx

```diff
@@ -51,7 +53,7 @@ const renderRecord = (record: any) => {
       resource = <div>IPV6 address: {record.address}</div>;
       break;
     case "TXT":
-      resource = <div>TXT: {record.txt}</div>;
+      resource = <div>TXT: <pre className="txt">{record.txt}</pre></div>;
```

Comment:
would you hate me if i think this formatting should be applied to all record types?